src/types/tokenstream.cpp

```diff
@@ -0,0 +1,604 @@
+/**
+ * @file src/types/tokenstream.cpp
+ * @brief Implementation of class Literal.
+ * @copyright (c) 2017 Avast Software, licensed under the MIT license
+ */
+
+#include <algorithm>
+#include <cassert>
+#include <sstream>
+#include <stack>
+
+#include "yaramod/types/tokenstream.h"
+#include "yaramod/utils/utils.h"
+
+namespace yaramod {
+
+#define TABULATOR_LENGTH 8
```

Comment:
I would turn preprocessor define into constexpr const value.

Reply:
That is a very good point, thank you. Fortunately, the macro didn't cause any memory errors, but it will be much better to use `constexpr` instead.